editor/inserter/menu.js

```diff
@@ -267,14 +264,48 @@ class InserterMenu extends Component {
 		);
 	}
 
-	switchTab( tab ) {
-		this.setState( { tab: tab } );
+	renderBlocks( separatorSlug, blocks ) {
+		const { instanceId } = this.props;
+		return (
+			<div
+				className="editor-inserter__category-blocks"
+				role="menu"
+				tabIndex="0"
+				aria-labelledby={ `editor-inserter__separator-${ separatorSlug }-${ instanceId }` }
```

Comment:
One issue here, which existed previously as well, is that for the "Recent" tab, we assign an attribute claiming that the block group is labelled by the separator, but there is no separator corresponding to the specified ID. You can see this in the Developer Tools Inspector by searching "editor-inserter__separator-recent-". You'll find only the `aria-labelledby` attribute, but no element with that ID exists.